client/src/components/Reviews.jsx

```diff
@@ -9,14 +9,26 @@ const Reviews = (props) => (
             <span>Dined on </span><span>{sampleReview.review_time}</span>
```

Comment:
do you need to reference the data as {props.sampleReviews.review_time}...I could be wrong.